moped-editor/package.json

```diff
@@ -75,6 +75,7 @@
     "html-react-parser": "^1.4.14",
     "lodash.foreach": "^4.5.0",
     "lodash.get": "^4.4.2",
+    "lodash.isequal": "^4.5.0",
     "lodash.toarray": "^4.4.0",
```

Comment:
This is a very low priority question / issue - this is the furthest from a regression or a show stopper.

Would it be worthwhile to bump up `lodash.get` to `^4.5.0` and `lodash.toarray` to `^4.5.0` as well? I think doing this would be a non-change, as as-written, the only common lodash versions that are usable are 4.5.0 - 4.17.21.

I have not used multiple versions of a library in a single package.json before, and it looks like it's supported, but it may reduce the burden of future upgrades if we only invoke one if we can get away with it. Can we bump up to 4.17.21? That last ask is squarely out of the scope of the issue too, no worries.